### PR TITLE
Corrections/additions to "CPU whitelist"

### PR DESCRIPTION
**Pull request by Zjaw**
I believe a comma was forgotten in the array from my last pull request (line 46). I don't know if that makes a difference or not in C++, but it still wasn't installing for me. I tried to correct that issue, reorganized the list, and added another requested CPU [(#944)](https://github.com/anbox/anbox/issues/944).

### CODE REVIEW

File: src/anbox/cmds/check_features.cpp [hunk 1]
```diff
@@ -30,22 +30,22 @@ std::vector<std::string> cpu_whitelist = {
...
   // Intel Xeon E5520
-  "E5520"
+  "E5520",
+  // Intel Pentium T4500
+  "T4500",
   // Intel Core2 Duo T6500
-  "T6500"
+  "T6500",
+  // Intel Core2 Duo T9600
+  "T9600"
```
Comment (morphis) [inline src/anbox/cmds/check_features.cpp]: According to http://www.cpu-world.com/CPUs/Core_2/Intel-Core%202%20Duo%20Mobile%20T9600%20AW80576GH0726M%20(BX80576T9600).html the CPU doesn't support SSE 4.2 so we can't add it here.

File: src/anbox/cmds/check_features.cpp [hunk 2]
```diff
@@ -30,22 +30,22 @@ std::vector<std::string> cpu_whitelist = {
...
-  "T4500", 
-  // Intel Core i7 Q720
-  "Q 720",
   // Intel Xeon E5520
-  "E5520"
+  "E5520",
+  // Intel Pentium T4500
+  "T4500",
   // Intel Core2 Duo T6500
-  "T6500"
```
Comment (morphis) [inline src/anbox/cmds/check_features.cpp]: Actually we have to drop this one too as it doesn't support SSE 4.2. See http://www.cpu-world.com/CPUs/Core_2/Intel-Core%202%20Duo%20Mobile%20T6500%20AW80577GG0452MA%20-%20AW80577GG0452ML.html

Comment (Zjaw) [reply src/anbox/cmds/check_features.cpp]: So SSE 4.1 is no longer viable to run anbox?

Comment (morphis) [reply src/anbox/cmds/check_features.cpp]: No, as stated in the code you need SSE 4.2 too. Did you run Anbox before with just SSE 4.1?

Comment (Zjaw) [reply src/anbox/cmds/check_features.cpp]: I was never able to get it to install. I thought at some point I had seen a thread or documentation stating that SSSE 3 and SSE 4.1 were supported, but if that is not correct than I do apologize. I was just hoping that whitelisting my CPU was all that was needed to get me up and running.

Comment (morphis) [reply src/anbox/cmds/check_features.cpp]: Nope, whitelisting wont help as long as the CPU doesn't support the necessary instructions.

**Pull request closed without merging**